solutionbox/structured_data/mltoolbox/_structured_data/_package.py

```diff
@@ -493,7 +507,8 @@ def cloud_train(train_dataset,
   job_request.update(dict(config._asdict()))
 
   if not job_name:
-    job_name = 'structured_data_train_' + datetime.datetime.now().strftime('%y%m%d_%H%M%S')
+    job_name = job_name_prefix or 'structured_data_train'
```

Comment:
Does this ever get to not having a job name, or do the public APIs always produce an appropriately named job_name? If yes, maybe mltoolbox_train instead?